pipeline-runner/tests/testthat/test-gem2s-2-load_user_files.R

```diff
@@ -443,3 +443,129 @@ test_that("parse_rhapsody_matrix uses RSEC if DBEC corrected counts are missing"
 
   expect_equal(values, expected_values)
 })
+
+
+test_that("read_10x_files removes rows with empty feature names both in count matrix and annotation if present and < 0.1%", {
+  counts <- mock_counts()[rep(seq_len(nrow(mock_counts())), each = 10), ]
+  rownames(counts)[2]=""
+  rownames(counts)[3]=".1"
+
+  features <- data.frame(
+    ensid = paste0("ENSFAKE", seq_len(nrow(counts))),
+    symbol = row.names(counts)
+  )
+  features[2:3,1:2] = ""
```

Comment:
we use arrows for assignment!
```suggestion
  features[2:3,1:2] <- ""
```